assets/wix/files.wxs

```diff
@@ -3153,6 +3147,57 @@
       <Component Id="cmp42A739F2AAC24D729DDBCAA096B8C55A">
         <File Id="fil48314E2AC49B4E208C5667CEC50CA78C" KeyPath="yes" Source="$(var.ProductSourcePath)\RegisterMicrosoftUpdate.ps1" />
       </Component>
+      <Component Id="cmp8AAE18271AD64A009017665757ED915B">
+        <File Id="filBD4DDF972FCA4F898D0ACABC42936DE6" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.PowerShell.SDK.pdb" />
```

Comment:
We haven't .pdb files in msi before so please remove all from the file.

Reply:
👍 

This is just what was built locally following the instructions for the _.wxs_ file.  Happy to remove things that aren't necessary.

Reply:
Still getting [build failure](https://dev.azure.com/powershell/PowerShell/_build/results?buildId=108371&view=logs&j=ec161491-f990-5215-209c-5fb381a7fcfb&t=fd903d37-c62b-5a02-a186-3af73ce70275&l=2258) on this file.

```
Current files to not match  {D:\a\1\PowerShell\assets\wix\Files.wxs}
```

Not a very descriptive error.  It details where in the build script it failed, but not what the mismatch was.